src/Microsoft.Health.Dicom.Api/DicomApiResource.Designer.cs

```diff
@@ -78,6 +78,15 @@ internal class DicomApiResource {
             }
         }
         
+        /// <summary>
+        ///   Looks up a localized string similar to Cannot specify value more than once..
```

Comment:
In the context, multiple values in `StringValues` means the query key is specified multiple times in query string, so `DuplicateValues` may not be a good present.
How about `DuplicateQueryKey`? also include key name so that customer can easily identify the problematic key.

Reply:
That makes sense